apps/aecore/src/aec_chain_state.erl

```diff
@@ -499,6 +529,7 @@ db_children(#node{} = Node) ->
      || Header <- aec_db:find_headers_at_height(Height + 1),
         aec_headers:prev_hash(Header) =:= Hash].
 
+db_node_has_sibling_blocks(#node{type = micro}) -> false;
```

Comment:
This is not true. You might have micro forks.

Reply:
Oh, but I also note that the function is not correctly implemented. Once it is fixed it should handle micro blocks just fine.

Reply:
There is no bug, I was too hasty.